.eslintignore

```diff
@@ -0,0 +1,13 @@
+# DHIS2 Platform
```

Comment:
Seems like `.gitignore` is used by default. However once I added this file, the files were not ignored anymore - so I just copied the `.gitignore` contents.